packages/components/addon/components/hds/alert/index.hbs

```diff
@@ -11,13 +11,72 @@
     </div>
   {{/if}}
 
-  <div class="hds-alert__text">
-    {{#if @title}}
-      <div class="hds-alert__title">{{@title}}</div>
-    {{/if}}
+  <div class="hds-alert__content">
+
+    <div class="hds-alert__text">
+      {{#if @title}}
+        <div class="hds-alert__title">{{@title}}</div>
+      {{/if}}
+
+      {{#if @description}}
+        <div class="hds-alert__description">{{@description}}</div>
+      {{/if}}
+    </div>
 
-    {{#if @description}}
-      <div class="hds-alert__description">{{@description}}</div>
+    {{#if @actions}}
```

Comment:
I haven't looked at any use cases in the "real world" of consumers yet, but I think that this is a really interesting approach! I can't recall seeing an Ember component that strongly recommends a default, but allows an escape hatch, in this manner